### ISSUE TITLE
[Feature Request] "Capture Devices" screen sharing option

### ISSUE THREAD

**Issue by ItsNiceCraft** [Vencord/Vesktop#302]
Discord recently introduced a "Capture Devices" option to screen sharing that allows you to directly stream the video and audio of a capture device such as an external capture card to a voice channel / call.

It'd be great to see this implemented in Vesktop, if possible.

![image](https://github.com/Vencord/Vesktop/assets/74358857/42812128-b527-44ce-be7f-cda880b386a1)

**Comment by Vendicated** [comment]
i don't think this is easily possible, since electron only supports screens & windows

try searching in the electron issues

**Comment by ItsNiceCraft** [comment]
Closing this then

**Comment by AAGaming00** [comment]
> i don't think this is easily possible, since electron only supports screens & windows
> 
> try searching in the electron issues

They're just cameras, you can access them the same way you would a webcam

the real issue is getting that into discord as a stream feed id imagine

**Comment by Vendicated** [comment]
no don't close, it's still useful to track, it's just important to note that it's likely not that simple

**Comment by ryanccn** [comment]
I have a draft here #195 but I would appreciate some help with the patching!

**Comment by rejedai** [comment]
![image](https://github.com/Vencord/Vesktop/assets/92713267/23b19f2f-007f-4438-996f-30fc7c120346)

Obs has an option to display cursor, is it possible to add similar in vesktop? Or is it also an electron limitation?

**Comment by Hexality** [comment]
Any update on this? 

Every time I screenshare with Vesktop, it somehow makes all my games laggy and in counterpart also uses a lotta ram. This doesn't happen when I stream using OBS, which means having a capture device connected to the virtual cam would probably make this less laggy.

**Comment by Novantric** [comment]
I wanna use Vesktop on Windows since I discovered it on Linux and this is the only real missing feature stopping me from using it full-time, as I tend to stream my cap card occasionally.

**Comment by makidoll** [comment]
Would be really nice to have this feature. Have been using OBS virtual camera with obs-vkcapture to reduce lag whilst capturing video games, but have to stream with webcam instead of screen share

**Comment by Fletch64** [comment]
Hello, I was curious if there was any update on this possibly? It's something I would love to be able to use.
